pennylane_forest/converter.py

```diff
@@ -0,0 +1,626 @@
+import warnings
+from collections.abc import Sequence
+
+import numpy as np
+import pennylane as qml
+import pennylane_forest as plf
+import pyquil
+import pyquil.gates as g
+
+pyquil_inv_operation_map = {
+    "I": qml.Identity,
+    "X": qml.PauliX,
+    "Y": qml.PauliY,
+    "Z": qml.PauliZ,
+    "H": qml.Hadamard,
+    "CNOT": qml.CNOT,
+    "SWAP": qml.SWAP,
+    "CZ": qml.CZ,
+    "PHASE": qml.PhaseShift,
+    "RX": qml.RX,
+    "RY": qml.RY,
+    "RZ": qml.RZ,
+    "CRX": qml.CRX,
+    "CRY": qml.CRY,
+    "CRZ": qml.CRZ,
+    "S": plf.ops.S,
+    "T": plf.ops.T,
+    "CCNOT": plf.ops.CCNOT,
+    "CPHASE": lambda *params, wires: plf.ops.CPHASE(*params, 3, wires=wires),
+    "CPHASE00": lambda *params, wires: plf.ops.CPHASE(*params, 0, wires=wires),
+    "CPHASE01": lambda *params, wires: plf.ops.CPHASE(*params, 1, wires=wires),
+    "CPHASE10": lambda *params, wires: plf.ops.CPHASE(*params, 2, wires=wires),
+    "CSWAP": plf.ops.CSWAP,
+    "ISWAP": plf.ops.ISWAP,
+    "PSWAP": plf.ops.PSWAP,
+}
+
+_control_map = {
+    "X": "CNOT",
+    "Z": "CZ",
+    "CNOT": "CCNOT",
+    "SWAP": "CSWAP",
+    "RX": "CRX",
+    "RY": "CRY",
+    "RZ": "CRZ",
+    "PHASE": "CPHASE",
+}
+
+_matrix_dictionary = pyquil.gate_matrices.QUANTUM_GATES
```

Comment:
This one seems to be redundant as ``ProgramLoader`` has this as an attribute as well (and that seems to be used unless there was some other intention for this one)

Reply:
Good catch!